<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureEvent_ShowGlobalAnimation</class>
 <widget class="QDialog" name="ConfigureEvent_ShowGlobalAnimation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Show Global Animation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui_icons.qrc">
    <normaloff>:/icons/openrpgmaker.png</normaloff>:/icons/openrpgmaker.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">.QFrame
{
	border-width: 1px;
	border-style: solid;
	border-color: palette(window-text);
}

.QFrame&gt;QLabel
{
	border-width: 0px;
	background-color: palette(window-text);
	color: palette(window);
	font-weight: bold;
}

.QFrame&gt;QCheckBox
{
	background-color: palette(window-text);
	border-width: 0px;
	border-style: none;
	color: palette(window);
	spacing: 3px;
	font-weight: bold;
	padding: 0px 0px 0px 3px;
}

.QFrame&gt;QCheckBox::indicator
{
	min-width: 8px;
	max-width: 8px;
	min-height: 8px;
	max-height: 8px;
	width: 8px;
	height: 8px;
	border-width: 2px;
	border-radius: 0px;
	border-style: ridge;
	border-color: palette(mid);
}

.QFrame&gt;QCheckBox::indicator:unchecked
{
	background-color: palette(base);
}

.QFrame&gt;QCheckBox::indicator:checked
{
	background-color: palette(window-text);
}

.QFrame&gt;QCheckBox::indicator:hover
{
	border-color: palette(highlight);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Action</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>3</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QRadioButton" name="rbPlayOnce">
                 <property name="text">
                  <string>Play Once</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbBeginPlaying">
                 <property name="text">
                  <string>Begin Playing</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbStopPlaying">
                 <property name="text">
                  <string>Stop Playing</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frmOption">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Option</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="margin">
                <number>3</number>
               </property>
               <item>
                <widget class="QCheckBox" name="cbHaltOtherProcesses">
                 <property name="text">
                  <string>Halt Other Processes</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frmGlobalAnimation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_293">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_218">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Global Animation</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_178" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_299">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="GlobalAnimationPreview" name="wGlobalAnimationPreview" native="true">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>120</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>160</width>
                <height>120</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cboxGlobalAnimation">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GlobalAnimationPreview</class>
   <extends>QWidget</extends>
   <header>globalanimationpreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gui_icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureEvent_ShowGlobalAnimation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureEvent_ShowGlobalAnimation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
